docs/chain-details/module_nft.md

```diff
@@ -0,0 +1,372 @@
+NFTs on application specific blockchains share some but not all features as their **Ethereum** brethren, 
+since application specific blockchains are more flexible in how their resources are utilized, such as the ability to use strings as IDs.
+
+The `nft` module here facilitates managing non-fungible tokens that represent individual assets with unique features on **Crypto.org Chain**.
+
+### Overview
+
+There are two key concepts for NFTs on **Crypto.org Chain**, namely, **denom** and **token**:
+
+- #### denom
+    A denom represents a collection of NFTs. 
+    For example, I could issue a denom named "CryptoPuppies" under which my collection of 100 CryptoPuppies NFTs get minted. 
+    Each denom has a `denom ID` and a `denom name`, both are unique on chain. 
+    A `denom schema` should generally be set when a denom gets issued, which indicates the format of NFT metadata under this denom.
+
+- #### token
+    An NFT, or simply "token", is a specific instance of NFT minted under a denom. Each token has a `token ID`, which is unique under a specific denom.
+    Generally, a token also has its `token name` (name of the NFT), 
+    `token URI` (off-chain information or storage location of the NFT), 
+    and `token metadata` (on-chain data that provides information about the NFT).
+
+::: tip Specifications
+`denom ID`: a string of lowercase alphanumeric characters with length between 3 and 64 that begins with a letter, unique over the chain;
+
+`denom name`: a non-empty string, unique over the chain;
+
+`denom schema`: a JSON metadata format for NFTs under this denom;
+
+`token ID`: a string that is unique under the denom;
+
+`token name`: a string;
+
+`token URI`: a string that directs to the off-chain information or storage location of the NFT;
+
+`token metadata`: a JSON object that matches the denom schema and represents the on-chain data that provides information about the NFT.
+:::
+
+Just as each user is uniquely identified by its address, each NFT is uniquely identified by the combination of its **denom ID** and its **token ID** (like a UID for the NFT),
+showing its uniqueness, non-interchangeability, and non-divisibility.
+
+---
+
+### Transactions and Queries
+
+### Transactions
+In general,
+1. any user may `issue` a denom as long as neither the **denom ID** nor the **denom name** has been taken;
+2. the creator of a denom, also know as the owner of the denom, is the only user who may `mint` an NFT under such denom;
+3. a user may `edit` or `burn` an NFT only if he/she is both the creator and the owner of that NFT;
+4. a user may `transfer` an NFT as long as he/she is the owner of that NFT.
+
+---
+
+### `issue`:
+Every NFT needs to "live" under a denom: an NFT collection. 
+Therefore, the first step is to issue a denom before one can mint NFTs:
+
+#### `tx nft issue [denom_id] --name [denom_name] --schema [denom_schema] --from [user_address]`- **Issue a denom**
+::: details Example: Issue a new denom with specified name and schema
```

Comment:
May add a paragraph break at the end. "Apart" attach in the same sentence.

Reply:
Removed "Apart ..." explanation from the code block section to the NOTE section just after it. Thanks.